test/jasmine/tests/click_test.js

```diff
@@ -47,7 +47,7 @@ describe('Test click interactions:', function() {
             setTimeout(function() {
                 click(x, y);
                 setTimeout(function() { resolve(); }, DBLCLICKDELAY / 2);
-            }, DBLCLICKDELAY / 4);
+            }, DBLCLICKDELAY / 2);
```

Comment:
There seem to be intermittent errors with click and doubleclick returning before the plot gets a chance to redraw. It would be better to ask the plot itself when it's done but I'm not sure how. We could try to tap into `gd._promises` but I'm not sure we'd know when the plot has *started* redrawing so that might be meaningless...